src/App.jsx

```diff
@@ -32,6 +36,10 @@ class App extends Component {
             <Route path="/articles/:slug" component={ViewArticle} />
             <Route path="/profile" component={ProfilePage} />
             <Route path="/social" component={SocialLogin} />
+            <Route path="/resetPassword" component={PasswordReset} />
+            <Route path="/updatepassword" component={UpdatePassword} />
+            <Route path="/passwordresetmessage" component={PasswordResetMessage} />
+            <Route path="/updatepasswordmessage" component={UpdatePasswordMessage} />
```

Comment:
In addition to @codeshifu's suggestion above, why do we need to have *four* different route paths for this feature?

Reply:
I need the extra 2 routes for redirecting the user to the login page after successfully updating their password and also interacting with the user when they input their email to update their password.

Reply:
I still do not think you need the extra routes (`passwordresetmessage` and `updateresetmessage`). You could come up with a modal or use a toastr component to display the feedback to the users.

Reply:
@okoroemeka You don't need `passwordresetmessage` and `updateresetmessage ` routes.
Please modify and implement changes.